src/core/unit.h

```diff
@@ -281,6 +281,10 @@ typedef struct Unit {
 
         BPFProgram *ip_bpf_ingress, *ip_bpf_ingress_installed;
         BPFProgram *ip_bpf_egress, *ip_bpf_egress_installed;
+        BPFProgramList *ip_bpf_custom_ingress;
+        BPFProgramList *ip_bpf_custom_ingress_installed;
+        BPFProgramList *ip_bpf_custom_egress;
+        BPFProgramList *ip_bpf_custom_egress_installed;
```

Comment:
should really bet `Set*` objects I am sure, see other comment

Reply:
Thanks :)